lib/sign_up.dart

```diff
@@ -0,0 +1,220 @@
+  @override
+  State<StatefulWidget> createState() {
+    return SignUpScreenState();
+  }
+}
+
+class SignUpScreenState extends State<SignUpScreen>{
+  TextEditingController _emailController = TextEditingController();
+  TextEditingController _passwordController = TextEditingController();
+  TextEditingController _confirmPasswordController = TextEditingController();
+
+
+  static Color textField_color2 = Color(0x42000000).withOpacity(0.05);
+  static Color textField_color = Colors.grey.withOpacity(0.10);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: ListView(
+
+        children: [
+          SizedBox(height: MediaQuery.of(context).size.height * 0.07,),
+          Stack(
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(left: 20,right: 20,top: 15,bottom: 10),
+                child: Container(
+                  child: Text('Get \nStarted',
+                  style: TextStyle(
+                    height: 1.1,
+                    fontWeight: FontWeight.bold,
+                    fontSize: 70,
+                    fontFamily: 'Poppins'
+                  ),),
+                ),
+              ),
+              Positioned(
+                top: 72,
+                  right: 75,
+                  child: Text('!',
+                    style: TextStyle(
+                        fontSize: 70,
+                        fontWeight: FontWeight.bold,
+                        fontFamily: 'Poppins',
+                        color: secondary_color
+                    )
+                    ,
+                  )
+              )
+            ],
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left: 20.0, right: 20.0),
+            child: Container(
+              child: Text('Sign Up and start saving!',
+                style: TextStyle(
+                  color: secondary_color,
+                  fontSize: 20,
+                    fontFamily: 'Poppins',
+                  fontWeight: FontWeight.bold
+                ),),
+            ),
+          ),
+          SizedBox(height: MediaQuery.of(context).size.height * 0.04,),
+          Padding(
+            padding: const EdgeInsets.only(left: 20.0, right: 20.0,bottom: 10),
+            child: Container(
+              child: Text('Email',
+                style: TextStyle(
+                  fontSize: 20,
+                    fontFamily: 'Poppins',
+                    fontWeight: FontWeight.bold
+                ),),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left:20.0,right: 20.0,bottom: 22.0),
+            child: Container(
+              height: MediaQuery.of(context).size.height*0.065,
+              child: TextFormField(
+                controller: _emailController,
+                onChanged: (value){
+
+                },
+                decoration: InputDecoration(
+                  labelText: 'Type something...',
```

Comment:
Can you please remove the labelText , it wont be useful as you have already given a header for the respective textformfield.
Similarily , kindly update the remaining two TextFormFields in the same way.

Reply:
Sure